logging/google/cloud/logging/handlers/__init__.py

```diff
@@ -19,3 +19,6 @@
     ContainerEngineHandler)
 from google.cloud.logging.handlers.handlers import CloudLoggingHandler
 from google.cloud.logging.handlers.handlers import setup_logging
+
+__all__ = ['AppEngineHandler', 'CloudLoggingHandler', 'ContainerEngineHandler',
```

Comment:
I'm very +1 on using `__all__`, as it tells users, sphinx, and pylint what the intended exports are for a package.